package.json

```diff
@@ -72,7 +72,8 @@
   },
   "workspaces": [
     "packages/@lwc/*",
-    "packages/lwc"
+    "packages/lwc",
+    "playground"
```

Comment:
Is there a reason to put `playground` at the top level? My preference is to name it as `@lwc/playground` (to avoid any attempted dependency confusion attacks) and put it in `packages`, similar to other internal private packages like `@lwc/integration-tests`.